conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -1,5 +1,3 @@
 # microservice specific routes
 
-POST        /v1/status/public-funds/nino/raw                @uk.gov.hmrc.homeofficesettledstatusproxy.controllers.testonly.HomeOfficeSettledStatusProxyTestOnlyController.statusPublicFundsByNinoRaw
-
 ->          /                                               prod.Routes
```

Comment:
can just remove this routes file if we want, assign the main router to just be prod.routes